# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/base/NotificationController.cpp
# Commit: 1e75ce1fef67
# Full Hash: 1e75ce1fef67f3139ac4f05993cad693c82c111c
# Author: Eitan Isaacson <eitan@monotonous.org>
# Date: 2020-02-17 09:50:03
# Regressor Bug: 1611060
# File Overlap Count: 1
# Description:
#   Bug 1611060 - Dispatch a11y scroll events for all DOM scrolls. r=Jamie
#   
#   Currently, only documents dispatch scroll events when in reality any
#   element can have scrollable overflows.
#   
# ==============================================================================

diff -r 20b59309d020 -r 1e75ce1fef67 accessible/base/NotificationController.cpp
--- a/accessible/base/NotificationController.cpp	Mon Feb 17 05:37:24 2020 +0200
+++ b/accessible/base/NotificationController.cpp	Mon Feb 17 04:09:34 2020 +0000
@@ -638,10 +638,6 @@
                  "isn't created!");
   }
 
-  // Initialize scroll support if needed.
-  if (!(mDocument->mDocFlags & DocAccessible::eScrollInitialized))
-    mDocument->AddScrollListener();
-
   // Process rendered text change notifications.
   for (auto iter = mTextHash.Iter(); !iter.Done(); iter.Next()) {
     nsCOMPtrHashKey<nsIContent>* entry = iter.Get();